lib/OpenQA/Worker/Job.pm

```diff
@@ -75,22 +75,23 @@ sub new {
         upload_results_interval   => undef,
         developer_session_running => 0,
     );
-    $self->{_status}                 = 'new';
-    $self->{_id}                     = $job_info->{id};
-    $self->{_info}                   = $job_info;
-    $self->{_livelog_viewers}        = 0;
-    $self->{_autoinst_log_offset}    = 0;
-    $self->{_serial_log_offset}      = 0;
-    $self->{_serial_terminal_offset} = 0;
-    $self->{_images_to_send}         = {};
-    $self->{_files_to_send}          = {};
-    $self->{_known_images}           = [];
-    $self->{_known_files}            = [];
-    $self->{_last_screenshot}        = '';
-    $self->{_current_test_module}    = undef;
-    $self->{_progress_info}          = {};
-    $self->{_engine}                 = undef;
-    $self->{_is_uploading_results}   = 0;
+    $self->{_status}                       = 'new';
+    $self->{_id}                           = $job_info->{id};
+    $self->{_info}                         = $job_info;
+    $self->{_livelog_viewers}              = 0;
+    $self->{_autoinst_log_offset}          = 0;
+    $self->{_serial_log_offset}            = 0;
+    $self->{_serial_terminal_offset}       = 0;
+    $self->{_images_to_send}               = {};
+    $self->{_files_to_send}                = {};
+    $self->{_known_images}                 = [];
+    $self->{_known_files}                  = [];
+    $self->{_last_screenshot}              = '';
+    $self->{_current_test_module}          = undef;
+    $self->{_progress_info}                = {};
+    $self->{_engine}                       = undef;
+    $self->{_is_uploading_results}         = 0;
+    $self->{_has_uploaded_logs_and_assets} = 0;
```

Comment:
I only added `_has_uploaded_logs_and_assets`. All the other lines are just tidy.

Reply:
I wish there was a way to make these less annoying to review... I end up counting the lines and looking for the new one...